core/build.gradle.kts

```diff
@@ -214,6 +219,9 @@ kotlin {
         val commonJava by creating {
             kotlin.srcDir("commonJava")
```

Comment:
Looks like I might have been the one smuggling that in, but I think this `srcDir` is not necessary.

Reply:
No problem. I've removed this.